hera_mc/cm_gen_sqlite.py

```diff
@@ -152,8 +169,8 @@ def update_sqlite(self, db_file="hera_mc.db"):
             f.write(inserts)
             f.write(".save {}\n".format(dbfile_full))
         subprocess.call("sqlite3 < {}".format(sqlfile), shell=True)
-        subprocess.call(f"rm -f {schema_file}{testtag}", shell=True)
-        subprocess.call(f"rm -f {inserts_file}{testtag}", shell=True)
+        subprocess.call(f"rm -f {schema_file}", shell=True)
+        subprocess.call(f"rm -f {inserts_file}", shell=True)
         subprocess.call(f"rm -f {sqlfile}", shell=True)
```

Comment:
I get these were always subprocess calls. but why not use like `os.remove` or Pathlib's `unlink` just to remove a file?

Reply:
done